lib/templates/structure/loop.php

```diff
@@ -14,90 +16,88 @@
  */
 do_action( 'beans_before_loop' );
 
-	if ( have_posts() && ! is_404() ) :
+if ( have_posts() && ! is_404() ) :
 
-		/**
+	/**
 		 * Fires before posts loop.
 		 *
 		 * This hook fires if posts exist.
 		 *
 		 * @since 1.0.0
 		 */
-		do_action( 'beans_before_posts_loop' );
+	do_action( 'beans_before_posts_loop' );
 
-		while ( have_posts() ) : the_post();
+	while ( have_posts() ) :
+		the_post();
 
-			$article_attributes = array(
-				'id'        => get_the_ID(), // Automatically escaped.
-				'class'     => implode( ' ', get_post_class( array( 'uk-article', ( current_theme_supports( 'beans-default-styling' ) ? 'uk-panel-box' : null ) ) ) ), // Automatically escaped.
-				'itemscope' => 'itemscope',
-				'itemtype'  => 'http://schema.org/CreativeWork',
-			);
+		$article_attributes = array(  // @codingStandardsIgnoreLine - WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedVariableFound.
+			'id'        => get_the_ID(), // Automatically escaped.
+			'class'     => implode( ' ', get_post_class( array( 'uk-article', ( current_theme_supports( 'beans-default-styling' ) ? 'uk-panel-box' : null ) ) ) ), // Automatically escaped.
+			'itemscope' => 'itemscope',
+			'itemtype'  => 'http://schema.org/CreativeWork',
+		);
 
-			// Blog specifc attributes.
-			if ( 'post' === get_post_type() ) {
+		// Blog specifc attributes.
+		if ( 'post' === get_post_type() ) {
 
-				$article_attributes['itemtype'] = 'http://schema.org/BlogPosting';
+			$article_attributes['itemtype'] = 'http://schema.org/BlogPosting'; // @codingStandardsIgnoreLine - WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedVariableFound.
 
-				// Only add to blogPost attribute to the main query,
-				if ( is_main_query() && ! is_search() ) {
-					$article_attributes['itemprop']  = 'blogPost';
-				}
+			// Only add to blogPost attribute to the main query.
+			if ( is_main_query() && ! is_search() ) {
+				$article_attributes['itemprop'] = 'blogPost'; // @codingStandardsIgnoreLine - WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedVariableFound.
 			}
+		}
 
-			beans_open_markup_e( 'beans_post', 'article', $article_attributes );
+		beans_open_markup_e( 'beans_post', 'article', $article_attributes );
 
-				beans_open_markup_e( 'beans_post_header', 'header' );
+			beans_open_markup_e( 'beans_post_header', 'header' );
 
-					/**
+				/**
```

Comment:
The rest of this comment needs to align left too to align with its opening `/**`.